app/serializers/user_serializer.rb

```diff
@@ -281,6 +282,10 @@ def ignored
     IgnoredUser.where(user_id: scope.user&.id, ignored_user_id: object.id).exists?
   end
 
+  def muted
+    MutedUser.where(user_id: scope.user&.id, muted_user_id: object.id).exists?
```

Comment:
Any chances this could introduce an N+1?

Reply:
We have this for `ignored` too. I think this recapping from our discussion about introducing similar query for `ignored`, this serializer seems to be used in the User profile but not `topics` for an instance (which is using a lighter serializer version).

Reply:
Might want to check that out

Reply:
![discourse____discourse_discourse__-_____lib_oneboxer_rb__discourse_](https://user-images.githubusercontent.com/45508821/54930926-0b598d00-4f10-11e9-853b-8d503abff111.png)

![Screenshot_25_03_2019__3_10_p_m_](https://user-images.githubusercontent.com/45508821/54931230-858a1180-4f10-11e9-9140-7ae6b197a240.png)

Reply:
I feel it is safe.